classes/PodsMeta.php

```diff
@@ -260,6 +263,7 @@ public function core() {
 				if ( ! pods_tableless() ) {
 					add_filter( 'add_user_metadata', array( $this, 'add_user_meta' ), 10, 5 );
 					add_filter( 'update_user_metadata', array( $this, 'update_user_meta' ), 10, 5 );
+					add_filter( 'update_user_metadata_by_id', array( $this, 'update_user_meta_by_id' ), 10, 4 );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 8c478ab2b7791d39f2ee04af0936c7df</sub>